apps/buildog/src/app/(auth)/layout.tsx

```diff
@@ -1,13 +1,12 @@
-import { Metadata } from "next";
+"use client";
 import Link from "next/link";
-import { SignUpForm } from "@/components/user-sign-up-form";
+import { ReactNode } from "react";
```

Comment:
In Next's project you don't need to import react, need to import it for the usage of the hooks and other compartments. This means that, you can directly give a type with React.ReactNode without `import { ReactNode } from "react";`

Reply:
You can check the suggestion at line 9